content/tutorials/custom-zk-chain/20.customizing-your-chain.md

```diff
@@ -0,0 +1,372 @@
+
+If your ERC20 token bridged to the L2 "acts" like ETH, then what happens if you bridge ETH to your custom chain?
+
+The answer is that if you try to bridge regular ETH to this chain, it will just use a different token contract address.
+
+You can try this out by depositing ETH from the L1 to this new chain:
+
+```bash
+npx zksync-cli bridge deposit \
+--rpc=http://localhost:3050 \
+--l1-rpc=http://localhost:8545
+```
+
+```shell
+? Amount to deposit 20
+? Private key of the sender 0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110
+? Recipient address on L2 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049
+```
+
+If you run the previous command to check the base token balance again, note how the balance is still `5`.
+Because ETH is no longer the base token, it has a different token address on L2 you will need to reference.
+To find the L2 token address for ETH, you can use the `l2TokenAddress` method available through `zksync-ethers`.
+
+To try this out, open the `zk-chain-test` folder from the previous section,
+and run the commands below to create a new script file:
+
+```bash
+mkdir scripts
+touch scripts/checkBalance.ts
+```
+
+Next, copy and paste the script below into the `checkBalance.ts` file:
+
+```ts
+import { ETH_ADDRESS_IN_CONTRACTS } from "zksync-ethers/build/utils.js";
+import { getWallet } from "../deploy/utils";
+
+async function main(){
+    const wallet = getWallet();
+    const l2ETHAddress = await wallet.l2TokenAddress(ETH_ADDRESS_IN_CONTRACTS);
+    console.log("L2 ETH Address:", l2ETHAddress);
+
+    const balance = await wallet.getBalance(l2ETHAddress);
+    console.log("L2 ETH Balance 🎉:", balance);
+}
+
+main();
+```
+
+Run the script with the `hardhat run` command:
+
+```bash
+npx hardhat run scripts/checkBalance.ts
+```
+
+You should see the output below showing the same amount of ETH you bridged:
+
+```shell
+L2 ETH Address: <0x_YOUR_ETH_TOKEN_ADDRESS>
+L2 ETH Balance 🎉: 20000000000000000000n
+```
+
+## Switching between chains and shutting down the ecosystem
+
+You can switch in between different chains without losing any state by shutting down the chain server and then running the command below:
+
+```bash
+zk_inception ecosystem change-default-chain
+```
+
+Now when you restart the server with `zk_inception server`, the new default chain's node will start again with the same state.
+
+To fully shut down the ecosystem and erase all of the data and state,
+you can shut down the Docker containers from the ecosystem folder using the command:
+
+```bash
+docker-compose down
+```
+
+To restart the ecosystem and run your custom ZK chain again, follow the steps below:
+
+1. In the ecosystem folder, run `zk_inception containers`.
+1. Redeploy your ERC20 contract to the L1.
+1. Update the base token address in `my_elastic_chain/chains/custom_zk_chain/configs/contracts.yaml` under `l1.base_token_addr` and in
+  `my_elastic_chain/chains/custom_zk_chain/ZkStack.yaml` under `base_token.address`.
```

Comment:
```suggestion
1. Update the base token address in `<my_elastic_chain>/chains/custom_zk_chain/configs/contracts.yaml` under `l1.base_token_addr` and in
  `<my_elastic_chain>/chains/custom_zk_chain/ZkStack.yaml` under `base_token.address`.
```

Reply:
I think this one if ok since I have that folder name in the setup, so the user doesn't need to change this